crypto/s2n_hash.c

```diff
@@ -96,6 +96,7 @@ static int s2n_low_level_hash_new(struct s2n_hash_state *state)
 
     state->is_ready_for_input = 0;
     state->currently_in_hash = 0;
+    memset(&state->digest,0,sizeof(state->digest));
```

Comment:
This works, but you could zero everything at once for completeness. Maybe just:
```suggestion
    *state = (struct s2n_hash_state) { 0 };
```
or 
```
memset(state, 0, sizeof(struct s2n_hash_state));
```